docs/pages/admin-guide.mdx

```diff
@@ -1188,9 +1188,9 @@ Environment="NO_PROXY=localhost,127.0.0.1,192.168.0.0/16,172.16.0.0/12,10.0.0.0/
   some other DNS name or a private IP address for it.
 </Admonition>
 
-## PAM Integration
+## PAM integration
 
-Review our dedicated [Using Teleport with PAM](features/ssh-pam.mdx) guide.
+Review our dedicated [Using Teleport with PAM](server-access/guides/ssh-pam.mdx) guide.
```

Comment:
This change is made in a previous Edit Pass that hasn't been merged. Added to pass the CI/CD checks.